functions/displayImage.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * gets element id from document and adds a background image to CSS
+ *
+ * @param {array} array takes an entered array
+ * @return {number} int takes id from html file and used as a key to get value from array and attach to string in new background image
+ */
+function displayImage (array, id) {
+	let card = document.getElementById(id)
```

Comment:
No point in defining a let here, it's just going to use up more memory. Just use document.getElementById().style

Reply:
Just a note, it's only ever worth defining a variable for a selector if you're going to be using it more than once.